### PR TITLE
feat: add auth pages ui

### PR DESCRIPTION
**Pull request by anik31**
## Auth Pages
I have added the auth pages i.e., login & signup. Ignore the index files. Focus only on the pages files. 

## Files to check
- [ ]  src/App.js
- [ ]  src/pages/auth/Login.jsx
- [ ]  src/pages/auth/Signup.jsx
- [ ]  src/pages/auth/auth.css
 
 _PS - Ignore other files._
 
## What not to check
- [ ]  src/pages/auth/index.js
- [ ]  src/pages/cart/index.js
- [ ]  src/pages/home/index.js
- [ ]  src/pages/index.js
- [ ]  src/pages/products/index.js
- [ ]  src/pages/wishlist/index.js

image
image

### CODE REVIEW

**Review by Developers-codz — APPROVED** [review]
Everything looks good to merge

**Comment by Dev-AkshatGupta** [comment]
This is really great man I m unable to find even one suggestion to be given to you .
Just wanted to ask What basically app.js helps u in achieving

**Comment by anik31** [comment]
> This is really great man I m unable to find even one suggestion to be given to you . Just wanted to ask What basically app.js helps u in achieving

@Dev-AkshatGupta app.js is the main component which is rendered by react. Inside it I'm fetching all the APIs to get the initial data (This will be noticeable when we have persisting data in the backend. Currently the backend doesn't persist data so nothing is shown on page refresh or 1st render). Then I have the routes according to which different pages are rendered.

File: src/pages/auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import "./auth.css";
+import {Link} from "react-router-dom";
+import {useState} from "react";
+
+export function Login(){
+    const [visibility, setVisibility] = useState(false);
+    
+    const showHideHandler = (e) => {
```
Comment (govindappaarun) [inline src/pages/auth/Login.jsx]: suggestion, can think of better name, eg: visibilityHandler

File: src/pages/auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { Link } from "react-router-dom";
+import "./auth.css";
+import {useState} from "react";
+
+export function Signup(){
+    const [visibility, setVisibility] = useState(false);
+    const [visibility1, setVisibility1] = useState(false);
```
Comment (govindappaarun) [inline src/pages/auth/Signup.jsx]: suggestion: naming, eg: isPasswordVisible, isConfPassWrodVisible

File: src/pages/auth/Signup.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                ? <i className="far fa-eye-slash"></i>
+                : <i className="far fa-eye"></i>}
+                </button>
+            </div>
+        </div>
+        <div className="input input-checkbox-radio">
+            <label><input type="checkbox" />I accept all terms & conditions</label>
+        </div>
+        <button className="btn btn-primary">Create New Account</button>
+        <Link to="/login" className="btn btn-secondary-icon-text-no-border">Already have an account <i className="fas fa-chevron-right"></i></Link>
```
Comment (govindappaarun) [inline src/pages/auth/Signup.jsx:61]: Suggestion: avoid long names, and not much reusable, try to compose the classes

File: src/App.js [hunk 4]
```diff
@@ -39,6 +39,8 @@ function App() {
           <Route path="/products" element={<Products />} />
```
Comment (govindappaarun) [inline src/App.js:39]: Your routes will go, suggestion: keep them in a separate file

**Review by govindappaarun — COMMENTED** [review]
I see some improvements in terms of naming, 
No major things, minor fixes

File: src/pages/auth/Login.jsx (hunk 1 above)
Comment (anik31) [reply src/pages/auth/Login.jsx]: Thank you for your suggestion, I have updated it.

File: src/pages/auth/Signup.jsx (hunk 2 above)
Comment (anik31) [reply src/pages/auth/Signup.jsx]: Thank you for your suggestion, I have updated it.

File: src/pages/auth/Signup.jsx (hunk 3 above)
Comment (anik31) [reply src/pages/auth/Signup.jsx:61]: Okay I will keep this in mind from next time.

File: src/App.js (hunk 4 above)
Comment (anik31) [reply src/App.js:39]: Can you give any reference ?

**Pull request merged**